### ISSUE TITLE
Kyma memory usage grows day by day towards higher end and it alarming for production deployment.

### ISSUE THREAD

**Issue by ankur-anand** [kyma-project/kyma#4714]
*Even When no services have been deployed kyma memory usage keep rising day by day*, and this has been an issue in past too.
This time to see if this is an issue I deployed Kyma 1.2 on Azure. and took the memory usage after deployment on day 1 and day 2 and today, No additional services have been deployed.

### Day1
```
kubectl top nodes && date
NAME                       CPU(cores)   CPU%   MEMORY(bytes)   MEMORY%
aks-nodepool1-54632527-0   439m         11%    2847Mi          22%
aks-nodepool1-54632527-1   517m         13%    2381Mi          18%
aks-nodepool1-54632527-2   634m         16%    3587Mi          28%
Wed Jun 18 11:15:18 IST 2019
```

### Day2
```
kubectl top nodes && date
NAME                       CPU(cores)   CPU%   MEMORY(bytes)   MEMORY%
aks-nodepool1-54632527-0   429m         10%    2993Mi          23%
aks-nodepool1-54632527-1   493m         12%    2617Mi          20%
aks-nodepool1-54632527-2   614m         15%    3922Mi          31%
Wed Jun 19 10:11:38 IST 2019
```

### Day15
```
kubectl top nodes && date
NAME                       CPU(cores)   CPU%   MEMORY(bytes)   MEMORY%
aks-nodepool1-54632527-0   542m         13%    4474Mi          35%
aks-nodepool1-54632527-1   419m         10%    3601Mi          28%
aks-nodepool1-54632527-2   661m         16%    5449Mi          43%
Tue Jul  2 10:28:56 IST 2019
```

**Comment by Demonsthere** [comment]
We have performed our own analysis, on two clusters in GKE:
- A full installation of kyma
- Installation without monitoring (prometheus, grafana, jaeger)

Full report here:
[kyma-performance-review.xlsx](https://github.com/kyma-project/kyma/files/3461102/kyma-performance-review.xlsx)

tl;dr version:

We have found that jaeger is the element with the biggest memory increase, followed by prometheus. As we though, the memory consumption is caused by elements that keep their data in a in-memory model. 
Second, lesser, thing we have found that several jobs in kyma are completed, but their pods persists - are not deleted after completing their tasks. This causes some extra resource reservations, that we could free

**Comment by piotrmsc** [comment]
CC @hisarbalik  I'm moving that to backlog huskies, I know upgrade of Prometheus is planned which may have an  impact so just FYI we have some finding described here :)

**Comment by lilitgh** [comment]
![Screenshot 2019-11-29 at 09 47 16](https://user-images.githubusercontent.com/29227648/69856206-64db3e00-128d-11ea-878a-c29a9e404780.png)

**Comment by a-thaler** [comment]
I checked the graph for memory consumption of nodes over one week having some re-occurring load. The consumption is not increasing over time and I assume that the problems are solved.
